config/owasp/suppressions.xml

```diff
@@ -38,7 +35,7 @@
 <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
 <vulnerabilityName>CVE-2020-9484</vulnerabilityName>
```

Comment:
we can replace tomcat I think. i did that

Reply:
yes ..i just testing ..not final one